prov/mrail/src/mrail.h

```diff
@@ -71,6 +75,11 @@ extern struct fi_fabric_attr mrail_fabric_attr;
 
 extern struct fi_info *mrail_info_vec[MRAIL_MAX_INFO];
 extern size_t mrail_num_info;
+extern int mrail_default_rail_small;
+extern int mrail_default_rail_large;
+extern int mrail_algorithm_small;
+extern int mrail_algorithm_large;
+extern int mrail_max_size_small;
```

Comment:
Maybe move these into a structure and/or use an array?  As with rxm, we may end up with more than just 2 options.

Reply:
That's a good idea.